src/Fixer/PhpUnit/PhpUnitAttributesFixer.php

```diff
@@ -196,6 +202,52 @@ private static function shouldBeFixed(string $annotationName, string $annotation
         return true;
     }
 
+    /**
+     * @param list<Token> $tokensToInsert
+     */
+    private static function isAttributeAlreadyPresent(Tokens $tokens, int $index, array $tokensToInsert): bool
+    {
+        $attributeIndex = $tokens->getNextMeaningfulToken($index);
+        if (!$tokens[$attributeIndex]->isGivenKind(T_ATTRIBUTE)) {
+            return false;
+        }
+
+        $insertedClassName = '';
+        foreach (\array_slice($tokensToInsert, 3) as $token) {
+            if ($token->equals('(') || $token->isGivenKind(CT::T_ATTRIBUTE_CLOSE)) {
+                break;
+            }
+            $insertedClassName .= $token->getContent();
+        }
+
+        // @TODO: refactor OrderedAttributesFixer::determineAttributeFullyQualifiedName to shared analyzer
+        static $determineAttributeFullyQualifiedName = null;
+        static $orderedAttributesFixer = null;
+        if (null === $determineAttributeFullyQualifiedName) {
+            $orderedAttributesFixer = new OrderedAttributesFixer();
+            $reflection = new \ReflectionObject($orderedAttributesFixer);
+            $determineAttributeFullyQualifiedName = $reflection->getMethod('determineAttributeFullyQualifiedName');
+            $determineAttributeFullyQualifiedName->setAccessible(true);
+        }
+
+        foreach (AttributeAnalyzer::collect($tokens, $attributeIndex) as $attributeAnalysis) {
+            foreach ($attributeAnalysis->getAttributes() as $attribute) {
+                $className = ltrim($determineAttributeFullyQualifiedName->invokeArgs(
```

Comment:
Since the method was marked as accessible, do we really need to use `invokeArgs()` on reflection, instead of just calling `$orderedAttributesFixer->determineAttributeFullyQualifiedName()`? Can't check it right now, reviewing from mobile 😅.

Reply:
Won't work:

> `Error: Call to private method PhpCsFixer\Fixer\AttributeNotation\OrderedAttributesFixer::determineAttributeFullyQualifiedName() from scope PhpCsFixer\Fixer\PhpUnit\PhpUnitAttributesFixer`